### ISSUE TITLE
ASWebAuthenticationSession in iOS 13 bug.

### ISSUE THREAD

**Issue by miguel-jimenez-0529** [okta/okta-oidc-ios#198]
Hello,

I'm currently facing an issue with the `pod 'OktaAuth', '~> 2.1'` on iOS 13.
When I start the signIn flow with:
```
OktaAuth
        .signInWithBrowser()
        .start(withDictConfig: config, view: self)
```
I receive the following error:
`Authorization Error: The operation couldn’t be completed. (org.openid.appauth.general error -3.)"`

After digging on the SDK I realize that the issue is caused because of the `ASWebAuthenticationSession` object. 
In iOS 13 `ASWebAuthenticationSession` has a property called `presentationContextProvider` which needs to be assigned in order to indicate which window the controller will be presented on.

I found the following [Article](https://dev.to/robotsquidward/quick-guide-to-aswebauthenticationsession-api-changes-in-ios-13-4m8i) indicating this additional step in iOS 13.

If you could indicate to me if this issue can be solved I would really appreciate it.

**Comment by IldarAbdullin-okta** [comment]
Hi @miguel-jimenez-0529 ,

Version 2.1 is not compatible with iOS 13. You have to install and start using 3.5.2 version

Ildar

**Comment by miguel-jimenez-0529** [comment]
Thanks for the quick response @IldarAbdullin-okta, I'm referring to the `pod 'OktaAuth'` not `pod 'OktaOidc'`. For `pod 'OktaAuth'` the latest version available is `2.1`.

**Comment by IldarAbdullin-okta** [comment]
`OktaAuth` is legacy pod and not supported anymore. Your only option is to start using `OktaOidc` 3.5.2
